### ISSUE TITLE
bug(fmt): does not work with Solidity v0.8.19 UDVT operators

### ISSUE THREAD

**Issue by PaulRBerg** [foundry-rs/foundry#4456]
### Component

Forge

### Have you ensured that all of these are up to date?

- [X] Foundry
- [x] Foundryup

### What version of Foundry are you on?

forge 0.2.0 (0453ac8 2023-03-01T00:13:03.658002Z)

### What command(s) is the bug in?

forge fmt

### Operating System

macOS (Apple Silicon)

### Describe the bug

Getting this error when running `forge fmt` in a project that uses the recently introduced [UDVT operators](https://blog.soliditylang.org/2023/02/22/user-defined-operators/):

```text
Error: ParserError
    ╭─[ValueType.sol:75:11]
    │
 75 │     H.add as +,
    ·           ─┬
    ·            ╰── unrecognised token 'as', expected "(", ")", ",", ".", ";", "case", "constant", "default", "error", "external", "for", "immutable", "internal", "leave", "override", "payable", "private", "public", "pure", "return", "returns", "revert", "switch", "view", "virtual", "{", "}", identifier
────╯
Error:
Failed to parse Solidity code for /Users/paulrberg/workspace/projects/prb-math/src/ud60x18/ValueType.sol. Leaving source unchanged.
```

For more context: https://github.com/PaulRBerg/prb-math/pull/168.

**Comment by rkrasiuk** [comment]
@mattsse not yet https://github.com/hyperledger/solang/pull/1203

**Comment by rkrasiuk** [comment]
closed by #4477
